plugins/webhooks/index.js

```diff
@@ -887,6 +891,8 @@ async function createEvents(
                 } catch (err) {
                     if (err.status >= 500) {
                         throw err;
+                    } else if (err.message !== undefined && err.message.indexOf('CircuitBreaker timeout') !== -1) {
```

Comment:
what's the status code for curcuit breaker errors ? perhaps we should define something at https://github.com/screwdriver-cd/circuit-fuses/blob/d99a1b3679803624f941eb8668ca2274eda37393/index.js to make that clear.

Also the `else` flow of simply logging seems to handle a valid scenario which is not well defined.

Reply:
You're right.
I'll change the way I fix it.

Reply:
@jithine 
Fixed by the following PR.
https://github.com/screwdriver-cd/circuit-fuses/pull/30